src/components/App.tsx

```diff
@@ -786,6 +787,17 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     this.scene.addCallback(this.onSceneUpdated);
     this.addEventListeners();
 
+    if (
+      "ResizeObserver" in window &&
+      this.excalidrawContainerRef?.current?.parentElement
+    ) {
+      this.resizeObserverContainer = new ResizeObserver(() =>
```

Comment:
nit: this is technically just a resizeObserver (instance), not a container, so I'd just call it `observer`.